test/integration/coarsegraining/coarsegraining_test.cpp

```diff
@@ -39,6 +39,9 @@
 // comparing the bilayer model with the baseline.
 #undef UPDATE_BASELINE
```

Comment:
So this makes it seem like the singleband model is just sitting around untested in the code.  Single_band chain should be removed or both should be tested. It cannot be pushed with changed code and no test.

Reply:
Ideally the test should run the two singleband and twoband models and compare the results. Unfortunately, there is no way to run two different models in the same execution without heavy refactoring of the cluster initializer. So I run the code with the flag 'UPDATE_BASELINE' producing an hdf5 file with the single-band result, and the test later runs without said flag comparing the twoband results with the baseline. I tried launching a separate process from the test to run the singleband, but I could not get it to work across systems. Do you have any idea on the best way to implement this sort of test?

Reply:
Thinking about your discussion with Urs its perhaps better to document this state of affairs an not introduce some epic hack around it.

Reply:
will do

Reply:
added description in a6bc040f3f854eae9be6e1982255c77a5d7e1bdc